reference/build_helpers/cmake.rst

```diff
@@ -292,3 +292,40 @@ Environment variables
 
 There are some environment variables that will also affect the ``CMake()`` helper class. Check them in the
 :ref:`CMAKE RELATED VARIABLES<cmake_related_variables>` section.
+
+Example
+-------
+The following example of ``conanfile.py`` shows you how to manage a project with conan and CMake.
+
+.. code-block:: python
+
+    from conans import ConanFile, CMake
+
+    class SomePackage(ConanFile):
+        name = "SomePackage"
+        version = "1.0.0"
+        settings = "os", "compiler", "build_type", "arch"
+        generators = "cmake"
+
+    def configure_cmake(self):
+        cmake = CMake(self)
+
+        # put definitions here so that they are re-used in cmake between
+        # build() and package()
+        cmake.definitions["SOME_DEFINITION_NAME"] = "On"
+
```

Comment:
Maybe here, insert the call to ``cmake.configure()``?

Reply:
Exactly :smile: added, sorry for the blind spot.